app/validators/has_one_title_validator.rb

```diff
@@ -0,0 +1,8 @@
+# validates that the title has at least one title
+class HasOneTitleValidator < ActiveModel::Validator
```

Comment:
@tpendragon is right. I believe this should be nuked: https://github.com/projecthydra-labs/curation_concerns/pull/716/files#diff-aba3d48be12193802ced467a01472193

The validator should live in `app/validators/`.